src/BsRecharts__PieChart.re

```diff
@@ -0,0 +1,47 @@
+open BsRecharts__Utils;
+
+/* http://recharts.org/en-US/api/PieChart */
+[@bs.module "recharts"]
+external reactClass : ReasonReact.reactClass = "PieChart";
+
+[@bs.obj]
+external makeProps :
+  (
+    ~height: int=?,
+    ~margin: margin=?,
+    ~onClick: (Js.t({..}), ReactEventRe.Mouse.t) => unit=?,
+    ~onMouseEnter: (Js.t({..}), ReactEventRe.Mouse.t) => unit=?,
```

Comment:
I think the first argument is nullable, here and in other handlers. Can you please check this and update bindings if needed?

Reply:
yes they are indeed, https://github.com/recharts/recharts/blob/3b807be36f35e0ae8ecb6918cd57f6c0bc257f4a/src/chart/generateCategoricalChart.js#L495 all them use this function getMouseInfo, except the onMouseLeave the first argument is not nullable in this one https://github.com/recharts/recharts/blob/3b807be36f35e0ae8ecb6918cd57f6c0bc257f4a/src/chart/generateCategoricalChart.js#L1029